src/constants.jl

```diff
@@ -1 +1,30 @@
+# Main constants
 const BOT_REPO_URL = "https://github.com/Humans-of-Julia/HoJBot.jl"
+
+const active_commands = LittleDict([
```

Comment:
I like moving all constants to constants.jl file.

Let's also start following Blue's recommendation for using ALL CAPS for global constants? https://github.com/invenia/BlueStyle#global-variables